indexer-app/src/cmr/indexer/data/concepts/collection.clj

```diff
@@ -49,6 +50,9 @@
       :else
       (errors/internal-error! (str "Unknown spatial representation [" coord-sys "]")))))
 
+(defn- update-values [m f & args]
```

Comment:
A docstring and more descriptive variable names would make this function a lot easier to read